msticpy/data/uploaders/splunk_uploader.py

```diff
@@ -59,11 +59,10 @@ def connect(self):
                     port=self.port,
                 )
             else:
-                logger.info("username/bearer_token are missing.")
-                logger.info("Credential loading from config file.")
+                logger.warning("Both 'username' and 'bearer_token' are missing.")
+                logger.info("Either param is necessary.")
```

Comment:
"One of these parameters must be supplied"
(not a big deal since it's a log message but I think this is clearer.

Actually, should this be an exception? a MsticpyConnectionError?
Seems that the connection has to fail at this point so not point in doing anything but throwing an exception.